packages/hierarchies-react/src/presentation-hierarchies-react/UnifiedSelectionContext.tsx

```diff
@@ -0,0 +1,62 @@
+import { createContext, PropsWithChildren, useContext, useEffect, useState } from "react";
```

Comment:
missing file header

Reply:
Added. Also updated script to handle `.tsx` files.